src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceSetDeserializer.cs

```diff
@@ -80,10 +81,11 @@ public sealed override object ReadInline(object item, IEdmTypeReference edmType,
             RuntimeHelpers.EnsureSufficientExecutionStack();
 
             IEdmStructuredTypeReference elementType = edmType.AsCollection().ElementType().AsStructured();
-
-            IEnumerable result = ReadResourceSet(resourceSet, elementType, readContext);
+            
+            IEnumerable result = ReadResourceSet(resourceSet, elementType, readContext);            
+             
             if (result != null && elementType.IsComplex())
```

Comment:
Up to you, but I would not insert a blank line here, as the condition immediately follows the assignment of result.